packages/react-components/src/tabs/src/TabList.tsx

```diff
@@ -21,11 +22,17 @@ const NavigationKeyBinding = {
     }
 };
 
+
+export interface TabListProps extends Omit<BoxProps, "autofocus"> {
```

Comment:
Not sure I understand what you are trying to do with this type definition by using `BoxProps` ?

Reply:
TabList does not have an as property. It also only uses 2 props, autoFocus and Tabs.

Any extra props passed will be forwarded to the Box component. This extends represents that. Any props that would be allowed on a box is allowed in this component. We omit the autoFocus, since the default autofocus only accepts a boolean, and we specify boolean | null

Reply:
Ok but this is the case for all our components, we basically always endup forwarding the props to a `<Box>` component and the other components interface aren't define like this.

Reply:
TabList now have a ref